js/views-manager.js

```diff
@@ -148,7 +148,8 @@ var ViewsManager = {
       });
       sortedFavedProjects.reverse(); // start from most recently favourited project
       // create a new div that shows count of favs
-      $("<div id='fav-count'><img src='svg/icon-fav.svg' width=20 />You have "+_.size(sortedFavedProjects)+" favs</div>").insertBefore(".projects");
+      var favs_count = _.size(sortedFavedProjects);
+      $("<div id='fav-count'><span class='star'></span> &nbsp; <span id='fav-count-text'>You have "+favs_count+" favs</div>").insertBefore(".projects");
```

Comment:
I'd rename the wrapper `div` to `fav-count-text` and have the inner `span` just wrap around the actual number count.

e.g., 
```html
You have <span id='fav-count'></span> favs.
```

then in `project-list-item.js` we could just update the fav number count instead of the full sentence (`$("#fav-count-text").html("You have "+favs_count+" favs");`)